packages/utils/src/constants/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const DEPOSIT_CONTRACT_TREE_DEPTH = 2 ** 5; // 32
+export const GENESIS_SLOT = 0;
+export const GENESIS_EPOCH = 0;
+export const GENESIS_START_SHARD = 0;
+export const FAR_FUTURE_EPOCH = Infinity;
+export const ZERO_HASH = Buffer.alloc(32, 0);
+export const ZERO_HASH_HEX = "0x" + "00".repeat(32);
+export const EMPTY_SIGNATURE = Buffer.alloc(96, 0);
+export const GRAFFITI_SIZE = 32;
+export const MAX_VARINT_BYTES = 10;
+
+/**
+ * The maximum milliseconds of clock disparity assumed between honest nodes.
+ */
+export const MAXIMUM_GOSSIP_CLOCK_DISPARITY = 500;
+export const MAXIMUM_GOSSIP_CLOCK_DISPARITY_SEC = MAXIMUM_GOSSIP_CLOCK_DISPARITY / 1000;
```

Comment:
The constants were moved in anticipation that they will be shared when the light-client is updated. But I agree it is premature in this PR. When working on the pr for the light-client, if they are actually needed then I will move it then and it will be obvious why.